src/central_control/full_system.launch

```diff
@@ -21,12 +21,14 @@
    <!-- Making navigation node required results in ROS shutting down after the entire path has been navigated.
         This is needed for automated testing. -->
   <node name="navigation" pkg="mapping_navigation" type="navigation.py" required = "true"/>
-  <node name="Ros_nav" pkg="Ros_nav" type="Ros_nav.py" />
```

Comment:
This script is gone. Can we just remove this entire line?

Reply:
Yes and I'll do it from my local

Reply:
Thanks!